toDoListApp/index.js

```diff
@@ -0,0 +1,134 @@
+'use strict';
+
+const taskInput = document.querySelector('.task-input');
+const taskList = document.querySelector('.collection');
+const clearBtn = document.querySelector('.clear-tasks');
+const form = document.querySelector('.create-task-form');
+
+const addBtn = document.querySelector('.btn');
+const containerBody = document.querySelector('.container');
+
+taskList.classList.add('font');
+containerBody.style.margin = '50px';
+addBtn.style.marginTop = '20px';
+
+
+const titles = document.getElementsByTagName('h3');
+for (let title of titles) {
+    title.classList.add('font');
+
+}
+
+form.addEventListener('submit', (event) => {
+    event.preventDefault();
+    if (taskInput.value.trim() === ''){
+        return;
+    }
+
+    createSingleTaskElement(taskInput.value);
+
+    storeTaskInLocalStorage(taskInput.value);
+
+    form.reset();
+})
+
+document.addEventListener('DOMContentLoaded', renderTasks);
+
+taskList.addEventListener('mouseover', (event) => {
+    let target = event.target.closest('li');
+    if(!target) return;
```

Comment:
```suggestion
    if(!target) {
        return;
    };
```
Не жалійте місце на нормальне форматування:)